db/seeds.rb

```diff
@@ -6,36 +6,36 @@
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
```

Comment:
Fix this file please, it's only the user initialization that you have to edit and should be good as new again.

Reply:
i guess it;s ok now